### DISCUSSION TITLE
VNC series example

### DISCUSSION THREAD

**Discussion by tholzheim** [SmartDataAnalytics/OpenResearch#181] [General]
Based on https://www.openresearch.org/wiki/VNC_2021 
* https://confident.dbis.rwth-aachen.de/or/index.php?title=VNC
* https://confident.dbis.rwth-aachen.de/or/index.php?title=Talk:VNC

**Comment by WolfgangFahl** [comment]
Indeed that is a good one - let's complete the series with provenance data from dblp, wikidata and wikicfp to show what we are up to. Tim could you please try this out with openrefine and avoid redundancy and stick to mandatory fields + provenance data ?

**Reply by tholzheim** [reply]
Started with:
![Screenshot from 2021-04-21 22-40-44](https://user-images.githubusercontent.com/45004115/115618421-f15b2600-a2f2-11eb-956a-a5d0b87f7d45.png)
Used ptp to complete the data:
Code example to extract the data from the json response
```python
import sys
sys.path.append("<Path to python site packages>")
import requests, json
url = requests.get(\"http://ptp.bitplan.com/parse?titles=\"+ value.replace(' ', '+') + \"&examples=example1&format=json&metadebug=on\")
text = url.text
data = json.loads(text)
property = "start_date"
property_value = None
for record in data.get("events"):
   if property in record:
         if property_value is None:
                  property_value = record.get(property)
                  return property_value
```
Ultimately it lead to the following result
```json
{
  "events" : [
    {
      "pageTitle" : "VNC 2009",
      "Acronym" : "VNC 2009",
      "Ordinal" : 1,
      "Title" : "2009 IEEE Vehicular Networking Conference (VNC)",
      "Homepage" : "https://ieee-vnc.org/2009/",
      "Start date" : "2009/10/28",
      "End date" : "2009/10/30",
      "Country" : "Japan",
      "City" : "Tokyo",
      "Series" : "VNC",
      "Type" : "Conference",
      "wikidataId" : null,
      "wikicfpId" : null
    },
    {
      "pageTitle" : "VNC 2010",
      "Acronym" : "VNC 2010",
      "Ordinal" : 2,
      "Title" : "2010 IEEE Vehicular Networking Conference (VNC)",
      "Homepage" : "https://ieee-vnc.org/2010/",
      "Start date" : "2010/12/13",
      "End date" : "2010/12/15",
      "Country" : "USA",
      "City" : "Jersey City",
      "Series" : "VNC",
      "Type" : "Conference",
      "wikidataId" : null,
      "wikicfpId" : null
    },
    {
      "pageTitle" : "VNC 2011",
      "Acronym" : "VNC 2011",
      "Ordinal" : 3,
      "Title" : "2011 IEEE Vehicular Networking Conference (VNC)",
      "Homepage" : "https://ieee-vnc.org/2011/",
      "Start date" : "2011/11/14",
      "End date" : "2011/11/16",
      "Country" : "The Netherlands",
      "City" : "Amsterdam",
      "Series" : "VNC",
      "Type" : "Conference",
      "wikidataId" : null,
      "wikicfpId" : 16146
    },
    {
      "pageTitle" : "VNC 2012",
      "Acronym" : "VNC 2012",
      "Ordinal" : 4,
      "Title" : "2012 IEEE Vehicular Networking Conference (VNC)",
      "Homepage" : "https://ieee-vnc.org/2012/",
      "Start date" : "2012/11/14",
      "End date" : "2012/11/16",
      "Country" : "Korea",
      "City" : "Seoul",
      "Series" : "VNC",
      "Type" : "Conference",
      "wikidataId" : "Q106335333",
      "wikicfpId" : 24590
    },
    {
      "pageTitle" : "VNC 2013",
      "Acronym" : "VNC 2013",
      "Ordinal" : 5,
      "Title" : "2013 IEEE Vehicular Networking Conference (VNC)",
      "Homepage" : "https://ieee-vnc.org/2013/",
      "Start date" : "2013/12/16",
      "End date" : "2013/12/18",
      "Country" : "USA",
      "City" : "Boston",
      "Series" : "VNC",
      "Type" : "Conference",
      "wikidataId" : "Q106335332",
      "wikicfpId" : 32284
    },
    {
      "pageTitle" : "VNC 2014",
      "Acronym" : "VNC 2014",
      "Ordinal" : 6,
      "Title" : "2014 IEEE Vehicular Networking Conference (VNC)",
      "Homepage" : "https://ieee-vnc.org/2014/",
      "Start date" : "2014/12/03",
      "End date" : "2014/12/05",
      "Country" : "Germany",
      "City" : "Paderborn",
      "Series" : "VNC",
      "Type" : "Conference",
      "wikidataId" : "Q106335331",
      "wikicfpId" : 41010
    },
    {
      "pageTitle" : "VNC 2015",
      "Acronym" : "VNC 2015",
      "Ordinal" : 7,
      "Title" : "2015 IEEE Vehicular Networking Conference (VNC)",
      "Homepage" : "https://ieee-vnc.org/2015/",
      "Start date" : "2015/12/16",
      "End date" : "2015/12/18",
      "Country" : "Japan",
      "City" : "Kyoto",
      "Series" : "VNC",
      "Type" : "Conference",
      "wikidataId" : null,
      "wikicfpId" : 44959
    },
    {
      "pageTitle" : "VNC 2016",
      "Acronym" : "VNC 2016",
      "Ordinal" : 8,
      "Title" : "2016 IEEE Vehicular Networking Conference (VNC)",
      "Homepage" : "https://ieee-vnc.org/2016/",
      "Start date" : "2016/12/08",
      "End date" : "2016/12/10",
      "Country" : "USA",
      "City" : "Columbus",
      "Series" : "VNC",
      "Type" : "Conference",
      "wikidataId" : null,
      "wikicfpId" : null
    },
    {
      "pageTitle" : "VNC 2017",
      "Acronym" : "VNC 2017",
      "Ordinal" : 9,
      "Title" : "2017 IEEE Vehicular Networking Conference (VNC)",
      "Homepage" : "https://ieee-vnc.org/2017/",
      "Start date" : "2017/11/27",
      "End date" : "2017/11/29",
      "Country" : "Italy",
      "City" : "Torino",
      "Series" : "VNC",
      "Type" : "Conference",
      "wikidataId" : "Q106335330",
      "wikicfpId" : null
    },
    {
      "pageTitle" : "VNC 2018",
      "Acronym" : "VNC 2018",
      "Ordinal" : 10,
      "Title" : "2018 IEEE Vehicular Networking Conference (VNC)",
      "Homepage" : "https://ieee-vnc.org/2018/",
      "Start date" : "2018/12/05",
      "End date" : "2018/12/07",
      "Country" : "Taiwan",
      "City" : "Taipei",
      "Series" : "VNC",
      "Type" : "Conference",
      "wikidataId" : null,
      "wikicfpId" : "123048"
    },
    {
      "pageTitle" : "VNC 2019",
      "Acronym" : "VNC 2019",
      "Ordinal" : 11,
      "Title" : "2019 IEEE Vehicular Networking Conference (VNC)",
      "Homepage" : "https://ieee-vnc.org/2019/",
      "Start date" : "2019/12/04",
      "End date" : "2019/12/06",
      "Country" : "USA",
      "City" : "Los Angeles",
      "Series" : "VNC",
      "Type" : "Conference",
      "wikidataId" : "Q106335329",
      "wikicfpId" : "155554"
    },
    {
      "pageTitle" : "VNC 2020",
      "Acronym" : "VNC 2020",
      "Ordinal" : 12,
      "Title" : "2020 IEEE Vehicular Networking Conference (VNC)",
      "Homepage" : "https://ieee-vnc.org/2020/",
      "Start date" : "2020/12/16",
      "End date" : "2020/12/18",
      "Country" : "Online",
      "City" : null,
      "Series" : "VNC",
      "Type" : "Conference",
      "wikidataId" : null,
      "wikicfpId" : 108826
    },
    {
      "pageTitle" : "VNC 2021",
      "Acronym" : "VNC 2021",
      "Ordinal" : 13,
      "Title" : "2021 IEEE Vehicular Networking Conference (VNC)",
      "Homepage" : "https://ieee-vnc.org/2021/",
      "Start date" : "2021/11/10",
      "End date" : "2021/11/12",
      "Country" : "Ulm",
      "City" : "Germany",
      "Series" : "VNC",
      "Type" : "Conference",
      "wikidataId" : null,
      "wikicfpId" : null
    }
  ]
}
```

**Reply by WolfgangFahl** [reply]
@tholzheim awesome work. This is the direction we'd love to go.

**Reply by Julian-tib** [reply]
Nice! 
Was the [series ](https://confident.dbis.rwth-aachen.de/or/index.php/VNC) completed automatically based on these results?

**Reply by tholzheim** [reply]
Based on the acronyms I extracted the missing information from wikidata and ptp and also generated some of them. For _Start date_ and _End date_ were more than half not queryable. I experimented a bit to extract them from the website but the early conferences had the dates as images and therefore I skipped the extraction form the website for the time being. Result was the json file as shown above. The json file was then used to generate the corresponding wikipages

**Comment by WolfgangFahl** [comment]
For each event we need the proceedings and TIBKat reference e.g. ISBN number / DOI and or TIBKat reference. 

https://dblp.org/db/conf/vnc/index.html has the ISBN numbers.

https://confident.dbis.rwth-aachen.de/or/index.php/VNC_2009

We have to discuss with the TIB about this 
![grafik](https://user-images.githubusercontent.com/1336221/115662547-6c4a2e00-a33f-11eb-80cc-fae42438d983.png)

**Comment by WolfgangFahl** [comment]
[TIB Katalog-Suche](https://www.tib.eu/de/suchen?tx_tibsearch_search%5Baction%5D=search&tx_tibsearch_search%5Bcnt%5D=20&tx_tibsearch_search%5Bcontroller%5D=Search&tx_tibsearch_search%5BDdocumentType%5D%5B0%5D=c&tx_tibsearch_search%5Bpg%5D=1&tx_tibsearch_search%5Bquery%5D=prefix%3A%28tibkat%29&cHash=7bc92b00eb26fe3e06d3ca33420d90a9)

um "VNC" Stichwort ergänzt führt zu:

![grafik](https://user-images.githubusercontent.com/1336221/115685901-f488fd00-a358-11eb-886c-44d54741b8bb.png)

Siehe https://rq.bitplan.com/index.php/TIB_Katalog

**Comment by WolfgangFahl** [comment]
https://kvk.bibliothek.kit.edu/?digitalOnly=0&embedFulltitle=0&newTab=0 ?

**Comment by Julian-tib** [comment]
TIBKAT: https://www.tib.eu/de/suchen/id/TIBKAT:68609493X/IEEE-Vehicular-Networking-Conference-VNC-2009-28?cHash=52ab98dd41bc360e786699012b220e0c
GND: https://lobid.org/resources/HT018563232 (only proceedings, the event is not entered in GND, however other events of this series are in GND e. g. https://lobid.org/gnd/16340512-8)
IEEE: https://ieeexplore.ieee.org/xpl/conhome/5410727/proceeding

**Reply by WolfgangFahl** [reply]
![grafik](https://user-images.githubusercontent.com/1336221/115690926-d7a2f880-a35d-11eb-9953-c1dd8fc09a81.png)

**Comment by Julian-tib** [comment]
[WikiData Entry 2009 IEEE Vehicular Networking Conference, VNC 2012, Tokyo, Japan, Octobre 28-30, 2009 ](https://www.wikidata.org/wiki/Q106591763)

**Comment by WolfgangFahl** [comment]
IFrame usage for CFPs? see https://rq.bitplan.com/index.php/Workdocumentation_2021-04-22

we only need the CFP url for this https://ieee-vnc.org/2021/cfp.html - we could copy the content and reference it via the frontend (as soon as we have one). And the legal fallout of this is also interesting. For me i just want the triple with subject vnc 2021 predicate has cfp link object https://ieee-vnc.org/2021/cfp.html

**Comment by tholzheim** [comment]
[IEEE Xplore](https://ieeexplore.ieee.org/xpl/conhome/1003110/all-proceedings) has the whole conference series with DOIs.
Is there any chance that we can get an [API Key](https://developer.ieee.org/).

**Reply by WolfgangFahl** [reply]
I'll check options.

**Comment by Julian-tib** [comment]
https://confident.dbis.rwth-aachen.de/or/index.php?title=Property_talk:ISBN

It seems that there is no central look up system for ISBN or a way to "resolve" ISBNs like one can with DOIs. Some ISBNs can only be found when searching directly in an indexing system like tib.eu or the publishers index. For example although 978-1-4244-5685-7 can be found in TIB KAT and at the publishers website, search engines (e. g. https://www.bookfinder.com/search/?isbn=978-1-424-45685-7&mode=isbn&st=sr&ac=qr) or websites intended to find a ISBN can't find it.

See https://www.doi.org/factsheets/ISBN-A.html for information about differences between ISBN and DOI.

Sticking to using Property:TIBKAT-ISBN might be our best chance to work with ISBNs. Julian Franken (talk) 12:50, 22 April 2021 (CEST)

**Reply by WolfgangFahl** [reply]
My main concerns is currently to get the biblographic references at all see #195
